rbs/SignatureTranslator.cc

```diff
@@ -19,7 +19,7 @@ rbs_string_t makeRBSString(const string &str) {
 unique_ptr<parser::Node>
 SignatureTranslator::translateAssertionType(vector<std::pair<core::LocOffsets, core::NameRef>> typeParams,
                                             const rbs::RBSDeclaration &assertion) {
-    string signatureString = assertion.string();
+    string signatureString = assertion.string;
```

Comment:
Let's not copy the string and either make this a `const string &` or, since we only use `signatureString` once in this function, just delete this variable and use `assertion.string` directly.